packages/pwa/app/pages/account/constant.js

```diff
@@ -52,18 +52,31 @@ export const navLinks = [
 ]
 
 export const CONFIRMATION_DIALOG_DEFAULT_CONFIG = defineMessages({
-    dialogTitle: {defaultMessage: 'Confirm Action'},
-    confirmationMessage: {defaultMessage: 'Are you sure you want to continue ?'},
-    primaryActionLabel: {defaultMessage: 'Yes'},
-    alternateActionLabel: {defaultMessage: 'No'}
+    dialogTitle: {defaultMessage: 'Confirm Action', id: 'global.account.title.confirm_action'},
+    confirmationMessage: {
+        defaultMessage: 'Are you sure you want to continue ?',
+        id: 'global.account.message.you_want_to_continue'
+    },
+    primaryActionLabel: {defaultMessage: 'Yes', id: 'global.account.action.yes'},
+    alternateActionLabel: {defaultMessage: 'No', id: 'global.account.action.no'}
 })
 
 export const REMOVE_CART_ITEM_CONFIRMATION_DIALOG_CONFIG = {
-    dialogTitle: defineMessage({defaultMessage: 'Confirm Remove Item'}),
+    dialogTitle: defineMessage({
+        defaultMessage: 'Confirm Remove Item',
+        id: 'global.account.title.confirm_remove'
```

Comment:
```suggestion
        id: 'cart_item.confirmation_dialog.title.confirm_remove'
```
Same with above, the name already indicates which component it belongs to, it makes sense to use it for the ids.